deploy/chart/local-path-storage-airgapped.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: v1
```

Comment:
It should be good enough to guide user how to modify the original deploy manifest instead of cloning another manfiest w/ less change for the template purpose.